src/spec.ts

```diff
@@ -87,12 +87,38 @@ const CAIP19AssetId: IdentifierSpec = {
   },
 };
 
+const BlockNumberTag: IdentifierSpec = {
+  name: "blockNumberTag",
+  regex: "[0-9]{1,78}|latest",
+  parameters: {
+    delimiter: "",
+    values: {
+      0: {
+        name: "blockNumberTag",
+        regex: "[0-9]{1,78}|latest",
+      },
+    },
+  },
+};
+
+const CAIP19BlockNumberTag: IdentifierSpec = {
```

Comment:
This seems to me not ideal but then again, IdentifierSpec isn't all that expressive make patterns...